src/gui/userinfo.cpp

```diff
@@ -123,13 +123,14 @@ void UserInfo::slotUpdateLastInfo(const QJsonDocument &json)
 
     // Quota
     auto objQuota = objData.value("quota").toObject();
+    qint64 quota = objQuota.value("quota").toDouble();
     qint64 used = objQuota.value("used").toDouble();
     qint64 total = objQuota.value("total").toDouble();
```

Comment:
This could have been all a one liner if we just changed `"total"` with `"quota"` here. As seen a few lines below this `total` variable is assigned to `_lastQuotaTotalBytes`. So it should hold the quota total not the storage total.

Reply:
I did this. I just hope I got it right.